internal/wrappers/cpp/kuzzle.cpp

```diff
@@ -123,6 +123,10 @@ namespace kuzzleio {
     return _listener_instances;
   }
 
+  void Kuzzle::emitEvent(Event event, const std::string& body) {
```

Comment:
Shall we document this method on the Kuzzle object ?

Reply:
This was not documented before so no I don't think we need to document it, it is supposed to be an internal method but I need to expose it in order to use it from the Auth controller.